package.json

```
@@ -41,17 +41,17 @@
         "through": "^2.3.8"
     },
     "devDependencies": {
-        "concat-stream": "^1.6.2",
+        "concat-stream": "^2.0.0",
         "eclint": "^2.8.1",
         "ecstatic": "^4.1.4",
         "eslint": "^6.8.0",
         "falafel": "^2.2.4",
         "js-yaml": "^3.13.1",
-        "tap": "^8.0.1",
-        "tap-parser": "^3.0.5"
+        "tap": "^14.10.7",
+        "tap-parser": "^10.0.1"
     },
     "scripts": {
-        "prelint": "eclint check",
+        "prelint": "eclint check index.js lib/* bin/* test/* example/*",
```

are those files not checked when no pattern is provided? also, would this work?
```suggestion
        "prelint": "eclint check '**/*.js'",
```

---

fixed in latest commit